Source/SteveSloth/Public/MyPlayer.h

```diff
@@ -116,50 +126,81 @@ class STEVESLOTH_API AMyPlayer : public ACharacter
 	UPROPERTY(EditAnywhere, Category = "Input|Actions")
 	UInputAction* PAiming;
 
+	UPROPERTY(EditAnywhere, Category = "Input|Actions")
+	UInputAction* PSwitchAbilities;
+
+	UPROPERTY(EditAnywhere, Category = "Input|Actions")
+	UInputAction* PCamPitch;
+
+	UPROPERTY(BlueprintReadOnly, EditAnywhere, Category = "Animations")
+	UAnimSequence* GrapplingHookSequence;
+
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, meta = (RowType = "MyAmmoData"), Category = "Data")
 	FDataTableRowHandle AmmoDataTable;
 
+	// You can expose some of your collision query data as properties to help customize and debug 
+	// Here we expose the collision channel we want to run the query on, and set it to only hit Pawns.
+	UPROPERTY(EditAnywhere, Category = "Collision")
+	TEnumAsByte<ECollisionChannel> TraceChannelProperty = ECC_Pawn;
+
+
+	FVector TraceEnd;
+	FVector TraceStart;
+	FVector WallHitLocation;
```

Comment:
I dont see where these three variables are used.. if they are cool, if not please remove